roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,153 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+configuration_directory: /opt/configuration
+
+##########################
+# k3s/metallb
+
+metallb_rook_external_IP: 192.168.16.101
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_ceph_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=quay.io/rook/ceph
+rook_ceph_image_tag: "v17.2"
```

Comment:
We started with Quincy because we deploy Quincy with ceph-ansible at the moment. The versions will be defined in osism/release in the future and the PR for 18.2.2 is already prepared: https://github.com/osism/release/pull/1402

Reply:
thanks for the explanation!

Reply:
We can bump the version here too, to not confuse the next one.

Reply:
Let's do this. I will merge the pending PRs in osism/release and we use the same versions here.